app/models/concerns/can_limit_number_of_people.rb

```diff
@@ -0,0 +1,81 @@
+module CanLimitNumberOfPeople
```

Comment:
Having looked at this I'm thinking of whether it would be worth (and whether it's possible) to treat this as a base for 2 distinct concerns `CanLimitNumberOfMainPeople` and `CanLimitNumberOfRelevantPeople`.

I think it would remove the need for a lot of code that returns a value based on a type; as distinct concerns they could just refer/use the value relevant to them.

What do you think?

Reply:
Fair point. I think I was trying to reduce duplication but it's probably just overcomplicated things! I'll have a bash at splitting them out again.